tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -97,6 +98,12 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
   @Input()
   yScaleType: ScaleType = ScaleType.LINEAR;
 
+  @Input()
+  customXFormatter?: Formatter;
```

Comment:
I don't see these used. Or is this what the TODO refers to? ("pipe the custom formatter to tooltip + axis renderer.").  It's a bit difficult to judge, as reviewer, whether this makes much sense - maybe consider deferring changes to line_chart_component to the future PR?

Reply:
Thanks for noting that. I forgot to write the glue code. Done.